externs/es5.js

```diff
@@ -16,7 +16,7 @@
 
 /**
  * @fileoverview Definitions for ECMAScript 5.
- * @see http://www.ecma-international.org/publications/files/drafts/tc39-2009-025.pdf
+ * @see http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-262.pdf
```

Comment:
That links to the latest edition which is currently the 6th. Although it's not as "official" I think it would be nice to link to https://es5.github.io/ since it has nice cross-reference links and such.